src/main.sh

```diff
@@ -12,11 +12,11 @@ main() {
   # Ensure env vars and args exist
   ensure::env_variable_exist "GITHUB_REPOSITORY"
   ensure::env_variable_exist "GITHUB_EVENT_PATH"
-  ensure::total_args 3 "$@"
```

Comment:
What do you think about checking if `TEAMWORK_API_TOKEN` exists as well? Isn't it mandatory as well?

Reply:
I think we don't need to check it because it's already defined as mandatory in the YML file.